assetsbrowser.py

```diff
@@ -95,9 +93,9 @@ def __init__(self, parent=None):
         self.splitter.setSizes([150, 500])
 
         # 4.4.2 Help Tab
-        html_file = 'ui/help/help.html'
-        temp_html_path = ('file:///' + str(repath(html_file)))
-        self.textBrowserHelp.setSource(QtCore.QUrl(temp_html_path))
+        absolute_path = os.path.abspath(os.path.dirname(__file__))
+        help_file = os.path.join(absolute_path, 'ui', 'help', 'help.html')
+        self.textBrowserHelp.setSource(QtCore.QUrl.fromLocalFile(help_file))
```

Comment:
`fromLocalFile` fix the relative path for image link in the HTML source code so no need hacky way of `repath`